<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LessonTextEditor</class>
 <widget class="QWidget" name="LessonTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>423</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_openTextFileButton">
       <property name="toolTip">
        <string>Open a text file...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_reformatLessonTextButton">
       <property name="toolTip">
        <string>Wrap lines automatically</string>
       </property>
       <property name="icon">
        <iconset theme="format-justify-fill"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_characterCountLabel">
       <property name="text">
        <string>0 Characters</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KTextEdit" name="m_lessonTextEdit">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_lessonTextEdit</tabstop>
  <tabstop>m_openTextFileButton</tabstop>
  <tabstop>m_reformatLessonTextButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
